File: II - Linguaggi di programmazione e laboratorio/Esame/Python/06_compito_java_20150917/Partita.py
class Partita:
    def __init__(self, tipo, data, ora, risultato, vinta):
        self._tipo = tipo
        self._data = data
        self._ora = ora
        self._risultato = risultato
        self._vinta = vinta
        
    def get_tipo(self):
        return self._tipo

    def __str__(self):
        return self._data + "\t" + str(self._ora) + "\t" + self._risultato

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/06_compito_java_2015_09_17/Partita.java
public class Partita {
    protected String data;
    protected String ora;
    protected String risultato;
    protected int vittoria;

    public Partita(String data, String ora, String risultato, int vittoria) {
        this.data = data;
        this.ora = ora;
        this.risultato = risultato;
        this.vittoria = vittoria;
    }

    public int getVinta() {
        return this.vittoria;
    }

}